src/uu/wc/wc.md

```diff
@@ -0,0 +1,9 @@
+# wc
+
+## Usage
```

Comment:
This line shouldn't be necessary anymore

Reply:
Hi @tertsdiepraam, I'm sorry it shows me the first three lines. Which one is redundant?

Reply:
removed